src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.ts

```diff
@@ -205,4 +209,15 @@ export class ExerciseDetailsStudentActionsComponent implements OnInit {
             }
         });
     }
+
+    importIntoIntelliJ() {
```

Comment:
From a testing point of view, this complicates things a lot more. The action component now has a complete seperate use case for communicating with Intellij.

Reply:
Refactored this into its own component